src/operator/OrgOverlay.tsx

```diff
@@ -57,6 +57,8 @@ const OrgOverlay: FC = () => {
   const history = useHistory()
   const canReactivateOrg =
     hasWritePermissions && organization?.state === 'suspended'
+  const isIOx = organization?.storageType == 'iox'
```

Comment:
for safety, let's convert `storageType` to lowercase before doing the comparison

```ts
organization?.storageType.toLowerCase() === 'iox'
```

note the `===` vs `==`, which forces the check to use [strict equality](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality) which won't do any type-casting. Not a huge deal when comparing strings, but it's more about the fact comparing using normal equality is often done explicitly to loosen the type checks. So in this case, it'll still do a string comparison just fine; the issue is that doing a non-strict check might make someone stop and see if there's a special reason not to do strict equality checking.